src/daft-connect/src/functions/aggregate.rs

```diff
@@ -0,0 +1,46 @@
+use daft_core::count_mode::CountMode;
+use daft_dsl::col;
+use daft_schema::dtype::DataType;
+use spark_connect::Expression;
+
+use super::{FunctionModule, SparkFunction, UnaryFunction};
+use crate::{error::ConnectResult, invalid_argument_err, spark_analyzer::SparkAnalyzer};
+
+pub struct AggregateFunctions;
+
+impl FunctionModule for AggregateFunctions {
+    fn register(parent: &mut super::SparkFunctions) {
+        parent.add_fn("count", CountFunction);
+        parent.add_fn("mean", UnaryFunction(|arg| arg.mean()));
+        parent.add_fn("stddev", UnaryFunction(|arg| arg.stddev()));
+        parent.add_fn("min", UnaryFunction(|arg| arg.min()));
+        parent.add_fn("max", UnaryFunction(|arg| arg.max()));
+        parent.add_fn("sum", UnaryFunction(|arg| arg.sum()));
+    }
+}
+
+struct CountFunction;
+impl SparkFunction for CountFunction {
+    fn to_expr(
+        &self,
+        args: &[Expression],
+        analyzer: &SparkAnalyzer,
+    ) -> ConnectResult<daft_dsl::ExprRef> {
+        match args {
+            [arg] => {
+                let arg = analyzer.to_daft_expr(arg)?;
+
+                let arg = if arg.as_literal().and_then(|lit| lit.as_i32()) == Some(1i32) {
+                    col("*")
```

Comment:
I believe any non-null literal should be supported

Reply:
so this is actually a bit different than supporting user provided `count(n)`. In spark you can only do `.count()` with no arguments. When this is passed over spark connect, it is hard coded as `count(1)`. There was a conversation on it [here](https://github.com/Eventual-Inc/Daft/pull/3677) and this code was just moved, not added.